crates/daphne/src/lib.rs

```diff
@@ -550,10 +545,9 @@ impl DapTaskParameters {
         &self,
         task_info: Vec<u8>,
         now: Time,
-        vdaf_verify_key_init: &[u8; 32],
-        collector_hpke_config: &HpkeConfig,
+        taskprov_config: roles::aggregator::TaskprovConfig<'_>,
     ) -> Result<(DapTaskConfig, TaskId, String), DapError> {
-        let taskprov_config = messages::taskprov::TaskConfig {
+        let task_config = messages::taskprov::TaskConfig {
```

Comment:
nit: I used `taskprov_config` instead of `task_config` for readability: `task_config` usually has type `DapTaskConfig`, so it makes it easier to disambiguate them.

Reply:
renamed to `taskprov_task_config`